manifests/job/present.pp

```diff
@@ -91,7 +93,7 @@
       command => "${cat_config} | ${update_job}",
       onlyif  => "test -e ${config_path}",
       unless  => "${difftool} ${config_path} ${tmp_config_path}",
-      require => File[$tmp_config_path],
+      require => $require,
```

Comment:
If this requires the `create-job`, then that implicitly also requires the config file.

Reply:
fixed it